fuse/config.c

```diff
@@ -209,14 +209,14 @@ static int config_check (SfsState* state) {
 static void config_init (SfsState* state) {
 	state->log_facility = -1;
 	state->update_mtime = UPDATE_MTIME_NO;
-	strcpy (state->hostname, "invalid");
+	strncpy (state->hostname, "invalid", sizeof state->hostname);
```

Comment:
i know but i does no harm either - it was said to be good practice to use this everywhere. strcpy is only available for compatibility reasons.